libnopegl/src/node_timerangefilter.c

```diff
@@ -30,107 +30,52 @@
 
 struct timerangefilter_opts {
     struct ngl_node *child;
-    struct ngl_node **ranges;
-    int nb_ranges;
+    double start_time;
+    double end_time;
+    double render_time;
     double prefetch_time;
-    double max_idle_time;
 };
 
 struct timerangefilter_priv {
-    int current_range;
     int drawme;
+    int updated;
```

Comment:
nit: maybe declare it before drawme (updates happen before draw).

Reply:
Sure, done